### PR TITLE
Updated manifests CRDs and temporary CSV

### PR DESCRIPTION
**Pull request by ppatierno**
Updated with final manifests CRDs for AMQ Streams 1.5
Updated temporary version of CSV file for AMQ Streams 1.5

### CODE REVIEW

File: common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -309,24 +329,18 @@ spec:
     Red Hat AMQ Streams provides a way to run an [Apache Kafka](https://kafka.apache.org) cluster on [OpenShift](https://www.openshift.com/) in various deployment configurations.
     See the Red Hat AMQ [website](https://access.redhat.com/products/red-hat-amq) for more details about the project.
 
-    ### What is new in AMQ Streams 1.4
-
-    * Add support for Keycloak authorization
+    ### What is new in AMQ Streams 1.5
 
-    * Add support for Mirror Maker 2.0
+    * Suppor for Apache Kafka 2.5.0 and 2.4.1
```
Comment (scholzj) [inline common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml:334]: In Streams we do not support 2.4.1

File: common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -309,24 +329,18 @@ spec:
...
 
-    * Add possibility to specify custom server certificates to TLS based listeners
+    * Built-in Cruise Control support for cluster rebalancing
```
Comment (scholzj) [inline common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml:336]: Should this say it is a Tech Preview?

File: common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -331,9 +331,9 @@ spec:
 
     ### What is new in AMQ Streams 1.5
 
-    * Suppor for Apache Kafka 2.5.0 and 2.4.1
+    * Suppor for Apache Kafka 2.5.0 and 2.4.0
```
Comment (scholzj) [inline common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml:334]: I would remove the 2.4.0 completely - new is only 2.5.0. In Strimzi both 2.4.1 and 2.5.0 were new in 0.18.0.

Comment (ppatierno) [reply common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml:334]: damn you are right, it's about what is new!

File: common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -309,24 +329,18 @@ spec:
     Red Hat AMQ Streams provides a way to run an [Apache Kafka](https://kafka.apache.org) cluster on [OpenShift](https://www.openshift.com/) in various deployment configurations.
     See the Red Hat AMQ [website](https://access.redhat.com/products/red-hat-amq) for more details about the project.
 
-    ### What is new in AMQ Streams 1.4
-
-    * Add support for Keycloak authorization
+    ### What is new in AMQ Streams 1.5
 
-    * Add support for Mirror Maker 2.0
+    * Suppor for Apache Kafka 2.5.0
```
Comment (scholzj) [inline common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml:334]:
```suggestion
    * Support for Apache Kafka 2.5.0
```

Sorry, I cannot stop :-o

Comment (ppatierno) [reply common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml]: If I make stupid mistakes it's not your fault :-)

Comment (scholzj) [reply common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml]: This is actually a typo in the upstream CSV already :-(

**Pull request merged**